molo/forms/wagtail_hooks.py

```diff
@@ -47,7 +45,22 @@ def create_new_page_relations(request, page, new_page):
                         relation.save()
 
 
-# This overrwrites the wagtailsuveys admin urls in order to use custom
+@hooks.register('construct_main_menu')
+def show_reactionquestions_response_for_users_have_access(request, menu_items):
+    excl = [
+        'reaction-question',
+        'reaction-question-responses',
+        'reaction-question-summary',
+    ]
+    menu_items[:] = [
+        item for item in menu_items if item.name not in excl]
```

Comment:
What is the purpose of this section above?

Reply:
@KaitCrawford remove the menu Items in the list from the main menu since they are subs of `reactionquestions`

Reply:
Do you mean this menu?
Without this code do they show up in the left column or the right column?
![cms_menu](https://user-images.githubusercontent.com/1230707/76881562-ff35b080-6881-11ea-9de8-369b0496eee2.png)

Reply:
@KaitCrawford yes